herald/apps.py

```diff
@@ -13,4 +16,32 @@ class HeraldConfig(AppConfig):
     name = 'herald'
 
     def ready(self):
+        from .models import Notification
+        from herald import registry
+
         self.module.autodiscover()
+
+        try:
+            # add any new notifications to database.
+            for index, klass in enumerate(registry._registry):
+                if klass.verbose_name:
+                    verbose_name = klass.verbose_name
+                else:
+                    verbose_name = re.sub(
+                        r'((?<=[a-z])[A-Z]|(?<!\A)[A-Z](?=[a-z]))',
+                        r' \1',
+                        klass.__name__
+                    )
+
+                notification, created = Notification.objects.get_or_create(notification_class=klass.__name__)
+                if created:
+                    notification.verbose_name = verbose_name
+                    notification.can_disable = klass.can_disable
+                    notification.save()
```

Comment:
These fields should update even if the notification was not just now created. Otherwise it would never change the verbose name or `can_disable` after the first time it is created.